src/dotnet/OpenAPI.Validator.Tests/Resource/OpenAPI/Validation/tracked-resource-patch-operation.json

```diff
@@ -20,8 +24,7 @@
         ],
         "operationId": "Redis_Get",
         "description": "gets a Redis cache.",
-        "parameters": [
-        ],
+        "parameters": [],
         "responses": {
           "200": {
```

Comment:
You should probably create a positive new test with a new test json which has `201` response as expected. This file should remain unchanged